### ISSUE TITLE
PWA Cache Worker can capture chrome-extensions traffic

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#96]
# Description
```
Please provide a short description of what you have encountered below.
```
It seems that chrome had compliant that the PWA cache worker is capable of caching chrome-extensions' request.  (looking at WebAnalyzer, uBlockOrigin, or PrivateBadger). This requires fixes as it threw an error out during use.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
No such error is raised in the future.

# Current Behavior
```
Please specify the current behavior of software below.
```
Occassionally, chrome may throw chrome-extensions request error.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```
1. Preview your site with F12 and observe console once in a while.

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Issue is similar to: https://stackoverflow.com/questions/68522967/failed-to-execute-put-on-cache-request-method-post-is-unsupported-pwa-s.

Attempting fixes.

**Comment by hollowaykeanho** [comment]
Fixed in eb8f22d1dac4b3fe6939040e6c6af01c79a32614

**Comment by hollowaykeanho** [comment]
Properly fixed in 59eb7e895cfaf921b0be205794fa12eef51b1e37

**Comment by hollowaykeanho** [comment]
Released in version `v.1.2.0`.
